src/V3Randomize.cpp

```diff
@@ -172,8 +172,11 @@ class RandomizeVisitor final : public VNVisitor {
     }
 
     AstCDType* findVlRandCDType(FileLine* fl, uint64_t items) {
+        int width = 0;
+        uint64_t tmp = items;
+        do { width++; } while (tmp >>= 1);
```

Comment:
Use clog2

But anyhow this is unrelated to this pull so please file a different one.  I don't see why needed changing - so need the test showing why too.

Reply:
Original `typeToHold` function has following signature.
```c++
    static string typeToHold(uint64_t maxItem) {
        return (maxItem < (1ULL << 8))    ? "CData"
               : (maxItem < (1ULL << 16)) ? "SData"
               : (maxItem < (1ULL << 32)) ? "IData"
                                          : "QData";
```
In last review, you suggested updating `typeToHold` to enable it to return `VlWide` to avoid duplicate code. But it's impossible to make maxItem bigger than `1ULL << 64` or something else and return `VlWide`. I have to change  argument `maxItem` into `width` and changed `typeToHold` is:
```c++
    static string typeToHold(int width) {
        if (width <= 8)
            return "CData";
        else if (width <= 16)
            return "SData";
        else if (width <= VL_IDATASIZE)
            return "IData";
        else if (width <= VL_QUADSIZE)
            return "QData";
        else
            return "VlWide<" + std::to_string(VL_WORDS_I(width)) + ">";
    }
```
Since the meaning of argument has been changed, I need to change every usage of original `typeToHold`. your commented code are for this purpose. Actually, it's the only user of original `typeToHold`.

> Use clog2

Do you mean `VL_CLOG2_Q` in `verilated_funcs.h`? But it's not proper to include `verilated_funcs.h` in source code?

Reply:
Ah, the change makes sense then, thanks.  You're right about VL_CLOG_2; V3Number::log2b would be the right function for Verilator sources.

Reply:
Since type of `maxItem` is `uint64_t` but `log2b` accepts `uint32_t` type. I have added new `V3Number::log2bl` function to accept `uint64_t` argument.